baip-0007.md

```diff
@@ -23,7 +23,7 @@ Participating assets: bitCNY, bitUSD, bitEUR, bitRUBLE
 2.Reward method:
 1)Participating accounts should register at the beginning of each cycle.Account does not participate in reward if not registered. The registration method can be to transfer any small number of bitassets to the specified account.
 2)The reward needs to be collected actively. If the reward is not collected within the time limit, this part of the reward will not be accumulated.The collection method can be to transfer any small number of bitasset to the specified account.
-3)If there are too few bitassets participating in the reward plan, set the upper limit of daily yield, such as 5 ‰.
+3)Set the upper limit of daily reward yield, such as 5 ‰. The purpose is to prevent the situation that the total amount of bitasset participating in the lock up award is too small and the yield is too high. Prevent waste of money.
 4)The more available bitassets you hold, the more rewards you get. Calculated as a minimum of 7 days. For example, if the number of bitCNY held within 7 days is 2000 3000 3000 1000 5000 5000 2000, then 1000 is the effective number. Note that the available bitassets do not include bitassets such as participating in Pending purchase\ compulsory liquidation and transfer.
 5)There is a reward for recommending new registered account to participate in lock in the reward plan. If the number of locked in bitassets in the account reaches a certain amount (e.g. 100000 bitcny, other bitassets are calculated based on the exchange rate year-on-year), it can be applied as a promotion account. If the new account recommended for registration by the account participates in the locked in of smart assets, the account can obtain corresponding rewards.
```

Comment:
>recommending new registered account to participate

Describe more clear if this involves built-in referral system of bitshares.

Reply:
This paragraph is only about how to reward. It is implemented by the organizer. There are two ways, one is to register the recommended account by the organizer, the other is to use the built-in recommendation system of the system. Of course, if the second way is used, the new recommended account can share the reward after a certain time. In summary, if this proposal is adopted, the organizer will formulate detailed implementation methods according to the requirements of this proposal.